### DISCUSSION TITLE
blog/second-post-with-postsimple

### DISCUSSION THREAD

**Discussion by giscus** [JoaomdvFerreira/d365-blog#3] [General]
# blog/second-post-with-postsimple

Testing post simple mechanics

http://localhost:3000/blog/second-post-with-postsimple

**Comment by AndreRodriguesSilva** [comment]
Why can't I tag you boy?

**Comment by AndreRodriguesSilva** [comment]
Weed to take a look on the look in feel of the posts, difficult to read...

**Reply by AndreRodriguesSilva** [reply]
Weed :D

**Reply by JoaomdvFerreira** [reply]
🌿🌿🌿🌿🌿🌿
